README.md

```diff
@@ -1,70 +1,183 @@
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+    <li><a href="https://www.postgresql.org/">Postgresql</a></li>
+    <li><a href="https://github.com/felixDev22/Final-capstone-backend.git">BACKEND[Ruby on Rails]</a><li>
+  </ul>
+  <summary>Client</summary>
+    <li><a href="https://github.com/felixDev22/Final-Capstone-Frontend.git">FRONTEND</a><li>
+</details>
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+<p align="right"><a href="#readme-top">front to top</a></p>
 
-### Code Splitting
+<!-- Key Features -->
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+## Key Features <a name="key-features"></a>
 
-### Analyzing the Bundle Size
+- Created an API endpint by rails.
+- Get the api from rails using cors
+- Created a react app
+- Used redux to manage the state
+- Used react router to navigate through the pages
+- Used react hooks to manage the state
+<!-- GETTING STARTED -->
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+## 💻 Getting Started <a name="getting-started"></a>
 
-### Making a Progressive Web App
+### Prerequisites
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+- Make sure you have Ruby installed in your system. You can install it [here](https://www.ruby-lang.org/en/documentation/installation/)
 
-### Advanced Configuration
+- Get started with [REACT](https://guides.rubyonrails.org/getting_started.html).
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+- Make sure you have [PostgreSQL](https://www.postgresql.org/) installed and running.
 
-### Deployment
+### Setup
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+- Clone this repository by running `git clone https://github.com/felixDev22/Final-Capstone-Frontend.git` in your command line.
 
-### `npm run build` fails to minify
+- Navigate to the repository by running `cd Final-Capstone-Frontend`.
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+### Install
+
+- Run `bundle install` to install all the gems.
+
+### Run
+
+- Run npm install
+- Run npm start
+- In your browser, go to `http://localhost:3000`.
```

Comment:
- [ ] You are instructing your users to run port 3000, but that one will be used by the server if it was run first.

Please, instruct your users to say "yes" when they run `npm install` and are asked if they want the frontend to be run on port 3001, which is default when the port 3000 is being used by another app (the server in this case).

After that, kindly tell your users to go to `http://localhost:3000`.

That way, your users will use the correct ports to run your app, and no confusion will take place with the backend port. 👍